source/library/Monadoc/Action/CronEntry/Enqueue.hs

```diff
@@ -44,19 +44,19 @@ runOne cronEntry = do
   pure (job, newCronEntry)
 
 enqueueJob :: CronEntry.Model -> App.App Job.Model
-enqueueJob = Job.Enqueue.run . CronEntry.task . Model.value
+enqueueJob = Job.Enqueue.run . (.task) . (.value)
```

Comment:
This syntax is unfortunately a bit clunky. It can be improved a bit though. I didn't realize you can chain multiple accessors in a single expression:
```suggestion
enqueueJob = Job.Enqueue.run . (.value.task)
```